modules/reward.js

```diff
@@ -60,8 +60,9 @@ function sendAndWriteReward(reward_type, transaction_id) {
 						`UPDATE ${tableName} SET reward_unit=?, reward_date=${db.getNow()} WHERE transaction_id=?`,
 						[unit, transaction_id],
 						() => {
-							let device = require('byteballcore/device.js');
-							device.sendMessageToDevice(row.device_address, 'text', `Sent the ${reward_type} reward`);
+							// commented out because already sending attestedSuccessFirstTimeBonus and referredUserBonus messages.
```

Comment:
These messages are sent when we are _going to_ send a reward, 'sent reward' is sent after it is actually sent, which can take time

Reply:
Yes, but this message is sent exactly same time as `Payment: x bytes`, so kind of redundant if a user already got `attestedSuccessFirstTimeBonus` or `referredUserBonus` message before.